R/shiny_vector_filter.R

```diff
@@ -120,18 +120,20 @@ shiny_vector_filter.default <- function(data, inputId, ...) {
 #'
 #' @return a pillar formatted class name
 #'
-#' @importFrom pillar new_pillar_type
-#' 
+#' @importFrom pillar type_sum
+#'
 get_dataFilter_class <- function(obj) {
   vf_methods <- gsub(".*\\.", "", as.character(methods(shiny_vector_filter)))
-  
+
   if ("numeric" %in% vf_methods)
     vf_methods <- c(vf_methods, "real", "double", "integer")
-  
+
   vf_class <- class(obj)
   vf_class <- vf_class[vf_class %in% vf_methods]
-  
+
   if (!length(vf_class)) return("unk")
   class(obj) <- vf_class
-  pillar::new_pillar_type(obj)[[1]][1]
+
+  type <- pillar::type_sum(obj)
+  if (length(type) == 0L) "" else type
```

Comment:
Hi @dgkf, did you intend to make this `{pillar}` change during this re-licensing? Or was this some old change not previously committed on your local branch? Just curious if it was intentional, that's all. Thanks!

Reply:
I did - this single line was the only code contributed by someone else. Although I was just clarifying the license, I wanted to be extra cautious. Assuming this is considered a full fledged relicensing, I had to either reach out to this individual and ask for permission to relicense their code or rewrite it myself. They don't seem to be very active on GitHub, so it was easier just to reimplement this one-liner. 

Coincidentally, I think this is a better way of doing what the line was doing before, but the primary purpose was to rewrite it myself to limit the impact of relicensing.